app/presenters/licences/set-up.presenter.js

```diff
@@ -0,0 +1,146 @@
+'use strict'
+
+/**
+ * Formats data for the `/licences/{id}/set-up` view licence set up page
+ * @module SetUpPresenter
+ */
+
+const { formatLongDate } = require('../base.presenter.js')
+
+const roles = {
+  billing: 'billing',
+  workflowEditor: 'charge_version_workflow_editor',
+  workflowReviewer: 'charge_version_workflow_reviewer'
+}
+
+/**
+ * Formats data for the `/licences/{id}/set-up` view licence set up page
+ *
+ * @returns {Object} The data formatted for the view template
+ */
+function go (chargeVersions, workflows, auth, commonData) {
+  return {
+    ..._authorisedLinks(auth, commonData),
+    chargeInformation: _chargeInformation(chargeVersions, workflows, auth)
+  }
+}
+
+function _authorisedLinks (auth, commonData) {
+  if (auth.credentials?.scope?.includes(roles.workflowEditor) && !_endsSixYearsAgo(commonData.ends)) {
+    return {
+      setupNewCharge: `/licences/${commonData.licenceId}/charge-information/create`,
+      makeLicenceNonChargeable: `/licences/${commonData.licenceId}/charge-information/non-chargeable-reason?start=1`
+    }
+  }
+
+  return {}
+}
+
+function _chargeInformation (chargeVersions, workflows, auth) {
+  return [
+    ..._workflows(workflows, auth),
+    ..._chargeVersions(chargeVersions)
+  ]
+}
+
+function _chargeVersions (chargeVersions) {
+  return chargeVersions.map((chargeVersion) => {
+    return {
+      id: chargeVersion.id,
+      startDate: chargeVersion.startDate ? formatLongDate(chargeVersion.startDate) : '-',
```

Comment:
A charge version must have a start date so we can drop the safe navigation here.

```suggestion
      startDate: formatLongDate(chargeVersion.startDate),
```